docs/pages/desktop-access/getting-started.mdx

```diff
@@ -18,6 +18,16 @@ The features documented on this page are available in Teleport `10.2.6` and high
 This guide will help you configure Teleport and Active Directory in order to
 provide secure, passwordless access to Windows desktops.
 
+<Details title="Compare Desktop Access to traditional RDP">
+
+Teleport Desktop Access is desiged to be a secure access protocol for Windows
+environments. Our protocol is designed with security as a priority, and may not
+be as performant as RDP or Guacamole. Consider Desktop Accesss to manage access
```

Comment:
Desktop Access speaks RDP, so maybe say "RDP clients like Guacamole"?

Reply:
Yeah, I wouldn't compare it to RDP, because it is RDP. I would compare it to more mature RDP _clients_. Teleport is not slower because of protocol, but because it is a rather simple RDP client and doesn't support advanced graphics pipelines, bitmap caching, etc.

Maybe say something like: Teleport implements a minimal feature set of the RDP protocol in order to reduce the surface area for potential attacks. This means it may not perform as well as more sophisticated RDP clients that support the full gamut of RDP features.

Reply:
I've adjusted the phrasing here.